_data/changelogs/component-file-input.yml

```
@@ -2,6 +2,17 @@ title: File input
 type: component
 changelogURL:
 items:
+  - date: 2023-05-06
```

```suggestion
  - date: NNNN-NN-NN
```

Keep as placeholder for now